core/src/main/java/nl/nn/adapterframework/jms/JMSFacade.java

```diff
@@ -734,7 +738,7 @@ public String getDestinationName() {
 	 * This function also sets the <code>useTopicFunctions</code> field,
 	 * that controls wether Topic functions are used or Queue functions.
 	 */
-	@IbisDoc({"either <code>queue</code> or <code>topic</code>", "<code>queue</code>"})
+	@IbisDoc({"2", "either <code>queue</code> or <code>topic</code>", "<code>queue</code>"})
 	public void setDestinationType(String type) {
 		this.destinationType = type;
```

Comment:
Zullen we hier een extra check op zetten dat je alleen QUEUE en TOPIC in kan/mag vullen (en het dan gelijk mooi oplossen door er een Enum van te maken?